operators/operator_sdk/osdk-pkgman-to-bundle.adoc

```diff
@@ -0,0 +1,14 @@
+[id="osdk-pkgman-to-bundle"]
+= Migrating package manifest projects to bundle format
+include::modules/common-attributes.adoc[]
+:context: osdk-pkgman-to-bundle
+
+toc::[]
+
+If you have an Operator project that was initially created using the package manifest format, which is deprecated, you can use the Operator SDK to migrate the project to the bundle format, which is the preferred packaging format for Operator Lifecycle Manager (OLM) starting in {product-title} 4.6.
+
+include::modules/osdk-about-pkg-format-migration.adoc[leveloffset=+1]
+.Additional resources
```

Comment:
This can be changed to `== Additional resources` with the `[id="additional-resources_"]` anchor because it's in an assembly, or moved into `modules/osdk-about-pkg-format-migration.adoc`.  😃

Reply:
Intention is for this to appear as if it's following the section, and not appear in the assembly's TOC.